src/Molecules/Filter/Filter.jsx

```diff
@@ -82,15 +87,36 @@ const InnerInput = ({
   )
 }
 
-const FilterWrapper = ({ children, onClear, ...props }) =>
-  <FlexBox minWidth="200px" flexDirection="column" px={2} py={1}>
-    <Filter {...props}/>
-    <Clear onClick={onClear}>
-      <Icon Component={Trash} size="small"/>
-      <Typo color="gray">
-        <Trans>global.action.remove</Trans>
-      </Typo>
-    </Clear>
-  </FlexBox>
+const FilterWrapper = ({
+  children,
+  onClear,
+  value: valueProp,
+  onChange: onChangeProp = () => {},
+  ...props
+}) => {
+  const [value, setValue] = useState(valueProp)
+  const onChange = () => value !== valueProp && onChangeProp(value)
+
+  return (
+    <FlexBox
+      onKeyPress={ev => ev.key === 'Enter' && onChange()}
+      minWidth="200px"
+      flexDirection="column"
+      px={2} py={1} gap={1}
+    >
+      <Filter value={value} setValue={setValue} onChange={onChange} {...props}/>
+      <Divider />
+      <FlexBox gap={1}>
+        <Action onClick={onClear}>
+          <Trash size={16} />

```

Comment:
Why not use `Icon` component ? This way we have the `small` (or whatever) size included

Reply:
`Icon` relies on the old theme that we are trying to remove